controllers/regex_test.go

```diff
@@ -27,19 +27,33 @@ func TestValidName(t *testing.T) {
 			Want: false,
 		},
 		{
-			Name: "nametoolong",
+			Name: "longname",
 			Args: args{
 				Name: "TestvalidNameTestvalidName",
 			},
-			Want: false,
+			Want: true,
 		},
 		{
-			Name: "maxlength",
+			Name: "max length",
```

Comment:
are spaces allowed in the client name?

Reply:
this is the test name rather. and no, spaces are not allowed